anomaly_detector/fact_store/fact_store_api.py

```diff
@@ -57,3 +59,13 @@ def readall_feedback(self):
         feedbacks = self.session.query(FeedbackModel).all()
         list = [f.to_dict() for f in feedbacks]
         return list
+
+    def readall_false_positive(self):
+        """Service for querying datastore of current false anomalies."""
+        items = self.session.query(FeedbackModel).all()
+        messages = set()
+        for i in items:
+            events = self.session.query(EventModel).filter_by(predict_id=i.predict_id)
```

Comment:
I did not review current version of the model, but I assume `predict_id` is primary key for EventModel and foreign key in FeedbackModel..you could probably use an SQL join and skip the for loop

Reply:
> I did not review current version of the model, but I assume `predict_id` is primary key for EventModel and foreign key in FeedbackModel..you could probably use an SQL join and skip the queries in th for loop

I'm planning on doing some changes to the schema. For now this works. I've opened a git issue based on your comment.